File: backendworker/NHSOnline.Backend.Users/Areas/Devices/Models/NotificationOutcomeResponse.cs
using System;
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;
using NHSOnline.Backend.Users.Areas.Devices.Models;

namespace NHSOnline.Backend.Users.Areas.Devices.Models
{
    public class NotificationOutcomeResponse
    {
        public string State { get; set; }

        public DateTime? EnqueueTime { get; set; }

        public DateTime? StartTime { get; set; }

        public DateTime? EndTime { get; set; }

        [SuppressMessage("Design", "CA1056:URI-like properties should not be strings")]
        public string PnsErrorDetailsUri { get; set; }

        public IEnumerable<PlatformOutcome> PlatformOutcomes { get; set; }
    }
}

File: bddtests/src/main/kotlin/worker/models/pushNotifications/NotificationOutComeResponse.kt
package worker.models.pushNotifications

data class NotificationOutComeResponse(val state:String,
                                       val enqueueTime:String,
                                       val startTime: String,
                                       val endTime: String,
                                       val pnsErrorDetailsUri: String,
                                       val platformOutcomes:List<PlatformOutCome>
                                    )
